dist/filelist

```diff
@@ -143,6 +143,9 @@ src/os_common/os_fstream.c
 src/os_common/os_fstream_stdio.c
 src/os_common/os_getopt.c
 src/os_common/os_strtouq.c
+src/os_darwin/os_futex.c         DARWIN_HOST
```

Comment:
Note our SConscripts do not have any parsing for DARWIN or LINUX tokens: https://github.com/mongodb/mongo/blob/8bc38915fc5cb5b389cc8f389eed7d8920d1684b/src/third_party/wiredtiger/SConscript#L174-L186

So there might need to be some adjustments on our side.

The bazel port of wiredtiger in mongo build has been delayed so the corresponding bazel filelist changes are fine and we can integrate those some time later.